src/jaxsim/rbda/contacts/relaxed_rigid.py

```diff
@@ -469,6 +561,16 @@ def continuing_criterion(carry: OptimizationCarry) -> jtp.Bool:
             )[0]
         )(position, velocity).flatten()
 
+        if n_kin_constraints > 0:
+            init_params = jnp.hstack(
+                [
+                    init_params,
+                    jnp.zeros(
+                        n_kin_constraints,
+                    ),
```

Comment:
Removing the traling comma might remove the additional newline
```suggestion
                    )
```

Reply:
Done in 2e43db70059d8fbc805e778036e7d1ed13022dc4